### PR TITLE
TDS2-2041 - Add home, mobile and work phone to card source

### PR DESCRIPTION
**Pull request by nick-kompas-cko**
Add home, mobile and work phone to card source

### CODE REVIEW

File: spec/components/schemas/Sessions/NetworkTokenSource.yaml [hunk 1]
```diff
@@ -33,11 +34,23 @@ allOf:
         enum:
           - vts
           - mdes
+          - applepay
+          - googlepay
+      cryptogram:
```
Comment (helder-goncalves-cko) [inline spec/components/schemas/Sessions/NetworkTokenSource.yaml]: We don't need `cryptogram` right? It doesn't fit in any of the AReq fields.

Comment (joey-griffin-cko) [reply spec/components/schemas/Sessions/NetworkTokenSource.yaml]: Won't the network token api need it?

Comment (nick-kompas-cko) [reply spec/components/schemas/Sessions/NetworkTokenSource.yaml]: a cryptogram isn't required for a 3ds2 (or 3ds1) authentication. 
It is only required for payments which isn't a sessions concern, right?

Comment (joey-griffin-cko) [reply spec/components/schemas/Sessions/NetworkTokenSource.yaml]: I'll remove it

File: spec/components/schemas/Sessions/NetworkTokenSource.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ allOf:
       - expiry_month
       - expiry_year
       - token_type
```
Comment (nick-kompas-cko) [inline spec/components/schemas/Sessions/NetworkTokenSource.yaml]: I don't see why the `token_type` might be needed?

**Comment by nick-kompas-cko** [comment]
LGTM but can't approve due to the initial commits being mine

File: spec/components/schemas/Sessions/SessionPhone.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+type: object
+description: A phone number
+properties:
+    cc:
+      description: Country code
```
Comment (helder-goncalves-cko) [inline spec/components/schemas/Sessions/SessionPhone.yaml]: should we call it `country_code`?

Comment (nick-kompas-cko) [reply spec/components/schemas/Sessions/SessionPhone.yaml]: yes this should be `country_code`

File: spec/components/schemas/Sessions/SessionPhone.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+      type: string
+      minLength: 1
+      maxLength: 3
+      example: 234
+    subscriber:
```
Comment (nick-kompas-cko) [inline spec/components/schemas/Sessions/SessionPhone.yaml]: this should now be 'number'.
It was changed after feedback in the PR that the sessions api doesn't have to adhere to the EMVco standard

**Comment by helder-goncalves-cko** [comment]
Merging this one and changes will be made in https://github.com/checkout/checkout-api-reference/pull/125

**Comment by nick-kompas-cko** [comment]
This was merged but the cc/subscriber fields are still wrong. Hopefully it's getting fixed in an upcoming PR?

**Pull request merged**